torchtitan/models/llama/__init__.py

```diff
@@ -30,6 +30,24 @@
     "debugmodel": TransformerModelArgs(
         dim=256, n_layers=8, n_heads=16, rope_theta=500000
     ),
+    "1B": TransformerModelArgs(
+        dim=2048,
+        n_layers=16,
+        n_heads=32,
+        n_kv_heads=8,
+        ffn_dim_multiplier=1.3,
```

Comment:
HF model definition is including `intermediate_size` directly, whereas torchtitan is using `ffn_dim_multiplier` to infer. Using
```suggestion
        ffn_dim_multiplier=1.4,
```
gives the right `intermediate_size=8192` and
> 982,386,688 total parameters